workflow-bot-app/src/test/java/com/symphony/bdk/workflow/CustomActivityIntegrationTest.java

```diff
@@ -19,6 +29,18 @@ void customActivity() throws Exception {
 
     // com.symphony.bdk.workflow.DoSomethingExecutor is just sending a message
     verify(messageService, timeout(5000)).send("123", "abc");
+
+    Response response = given()
```

Comment:
I'm wondering if we should add tests here or add custom activity based tests in the monitoring api tests class.

Reply:
It might not make a big difference, it is a matter of organisation. IMO, the custom activity is quite specific, I prefer to put related stuff in one place.